### PR TITLE
[zh] sync missing period in logging.md

### PR DESCRIPTION
**Pull request by Arhell**
sync PR https://github.com/kubernetes/website/pull/25435

### CODE REVIEW

**Comment by howieyuen** [comment]
I don't see any changes in the linking address...
what do I missing?

File: content/zh/docs/concepts/cluster-administration/logging.md [hunk 1]
```diff
@@ -159,7 +159,7 @@ up logging for COS image on GCP in the corresponding [script][cosConfigureHelper
 -->
 例如，你可以找到关于 `kube-up.sh` 为 GCP 环境的 COS 镜像设置日志的详细信息，
 相应的脚本在
-[这里](https://github.com/kubernetes/kubernetes/blob/{{< param "githubbranch" >}}/cluster/gce/gci/configure-helper.sh)
+[这里](https://github.com/kubernetes/kubernetes/blob/{{< param "githubbranch" >}}/cluster/gce/gci/configure-helper.sh).
```
Comment (tengqm) [inline content/zh/docs/concepts/cluster-administration/logging.md:162]: In Chinese, the period is "。"

Comment (Arhell) [reply content/zh/docs/concepts/cluster-administration/logging.md]: Done 
Thanks

**Pull request merged**